Source/JavaScriptCore/llint/InPlaceInterpreter64.asm

```diff
@@ -5955,12 +6028,15 @@ _wasm_ipint_call_return_location_wide32:
     loadi [MC], sc3
     advanceMC(IPInt::CallReturnMetadata::resultBytecode)
     leap [sp, sc3], sc3
-    addp FirstArgumentOffset - CallerFrameAndPCSize, sp
 
     const mintRetSrc = sc1
     const mintRetDst = sc2
 
-    move sp, mintRetSrc
+    loadi [MC], sc1
+    advanceMC(4)
+    leap [sp, mintRetSrc], mintRetSrc
```

Comment:
Isn't this just addp?

Reply:
It should be an `addp`, but the LLInt doesn't like using `addp` with `sp` on ARM, so I'm replacing it with an equivalent `leap` to keep it as one instruction (and not increase register pressure).